From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Prayoga <aditya@kobol.io>
Date: Tue, 15 Sep 2020 20:04:22 +0700
Subject: arm64: dts: rockchip: helios64: Armbian names for the system LEDs

Rename the status/fault LEDs: node system-gpio-leds, label
helios64::status (U-Boot board code and helios64-heartbeat-led.service
look it up), panic trigger on the fault LED, one pin group for both.

Upstream: none (Armbian naming).

Split out of add-board-helios64.patch (Aditya Prayoga, rebased onto
mainline by rpardini for 6.3); the DTB content is unchanged.

Signed-off-by: Aditya Prayoga <aditya@kobol.io>
Assisted-by: LLM
Signed-off-by: Igor Velkov <iav@iav.lv>
---
 arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts | 22 +++++-----
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
@@ -173,20 +173,23 @@ usb3 {
 		};
 	};
 
-	leds {
+	system_leds: system-gpio-leds {
+		status = "okay";
 		compatible = "gpio-leds";
 		pinctrl-names = "default";
-		pinctrl-0 = <&sys_grn_led_on &sys_red_led_on>;
+		pinctrl-0 = <&system_led>;
 
-		led-0 {
-			label = "helios64:green:status";
+		status-led {
+			label = "helios64::status";
 			gpios = <&gpio0 RK_PB4 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "none";
 			default-state = "on";
 		};
 
-		led-1 {
+		fault-led {
 			label = "helios64:red:fault";
 			gpios = <&gpio0 RK_PB5 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "panic";
 			default-state = "keep";
 		};
 	};
@@ -578,12 +581,9 @@ gphy_reset: gphy-reset {
 	};
 
 	leds {
-		sys_grn_led_on: sys-grn-led-on {
-			rockchip,pins = <0 RK_PB4 RK_FUNC_GPIO &pcfg_pull_down>;
-		};
-
-		sys_red_led_on: sys-red-led-on {
-			rockchip,pins = <0 RK_PB5 RK_FUNC_GPIO &pcfg_pull_down>;
+		system_led: sys-led {
+			rockchip,pins = <0 RK_PB4 RK_FUNC_GPIO &pcfg_pull_down>,
+					<0 RK_PB5 RK_FUNC_GPIO &pcfg_pull_down>;
 		};
 
 		network_act: network-act {
-- 
Armbian

